app/services/transaction_service/transaction.rb

```diff
@@ -6,19 +6,21 @@ module TransactionService::Transaction
 
   SETTINGS_ADAPTERS = {
     paypal: TransactionService::Gateway::PaypalSettingsAdapter.new,
-    braintree: TransactionService::Gateway::BraintreeSettingsAdapter.new,
     none: TransactionService::Gateway::FreeSettingsAdapter.new
   }
 
   GATEWAY_ADAPTERS = {
     paypal: TransactionService::Gateway::PaypalAdapter.new,
-    braintree: TransactionService::Gateway::BraintreeAdapter.new,
-    none: TransactionService::Gateway::FreeAdapter.new
+    none: TransactionService::Gateway::FreeAdapter.new,
+
+    # Deprecated adapters.
+    # We need these to be here, otherwise reading the old transactions will fail
+    checkout: TransactionService::Gateway::FreeAdapter.new,
+    braintree: TransactionService::Gateway::FreeAdapter.new,
```

Comment:
Did you think about solving this at the origin of the failure instead of providing fake adapters?